projects/assets-library/assets/styles/_color-palette.scss

```diff
@@ -84,6 +84,8 @@ $cloud-3: #a1c5e0;
 
 $brown-1: #9e4c41;
 
+$grey-1: #e5e6e6;
```

Comment:
We already have these colors below. The color you shared doesn't belong to this list. @michael-traceable Could you please chime in here?
```
$gray-1: #f4f5f5;
$gray-2: #e1e4e5;
$gray-3: #b7bfc2;
$gray-4: #889499;
$gray-5: #657277;
$gray-6: #40474a;
$gray-7: #272c2e;
$gray-8: #171a1c;
$gray-9: #080909;
```

Reply:
These look correct, yes.

Reply:
Actually, $gray-6 should be #3F474A. I wonder where that discrepancy came from

Reply:
@anandtiwary @jake-bassett do we want to make this change as well?
So this `$gray-6` is being used at many places.

Reply:
we have $gray-6 used at about 7-8 components today. we might want to review them before updating it. might be worth doing as a separate PR.

Reply:
Yeah, let's run with what we feel is correct as we know it today. If we break some colors in the mean time, that's fine. They weren't necessarily correct to begin with and its not worth blocking this PR for.

Let's create a ticket to do a follow up audit on those as @arjunlalb suggests. Do you mind doing that work next, @itssharmasandeep? Should be quick.

Reply:
Yeah, I can pickup this.